src/it/scala/io/iohk/ethereum/sync/RegularSyncItSpec.scala

```diff
@@ -28,24 +28,6 @@ class RegularSyncItSpec extends FreeSpecBase with Matchers with BeforeAndAfterAl
     testScheduler.awaitTermination(120.second)
   }
 
-  "a peer should reorganise when receives a checkpoint older than the current best from a peer" in customTestCaseResourceM(
```

Comment:
no longer needed? or abandoned due to checkpointing being put on hold

Reply:
the functionality that was inserting received checkpoint into waiting queues when the checkpoint is not on top was removed. Hence the changes in tests. But I can also bring all those deletions back

Reply:
I don't know anything about the new checkpointing, but if it's similar - I can revert all removals done for simplification, so we keep the old checkpointing features until we replace it later

Reply:
resolving, test is too specific to current implementation